.travis.yml

```diff
@@ -31,7 +31,5 @@ before_script:
 script: ./test.sh
 
 script:
-  - yarn
-  - yarn build:all
-  - yarn test
-  - cd backend && cargo test
+  - ./scripts/travis-backend.sh
```

Comment:
I think it would be easier to use npm/yarn scripts here.
For instance a new `yarn build:frontend` that, in this case, only goes into the right package and invoke yarn build.